README.md

```diff
@@ -1,109 +1,47 @@
-![cf](https://i.imgur.com/7v5ASc8.png) Lab 07: Vanilla HTTP Server
-======
+# HTTP Server
```

Comment:
your readme should tell the user specifically how to use your application. eg- how does one make the cow say stuff?